frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -216,12 +216,15 @@ export const PaymentInput = (): JSX.Element => {
             <FormControl
               isReadOnly={paymentsMutation.isLoading}
               isInvalid={!!errors.description}
+              isRequired
             >
               <FormLabel>Description</FormLabel>
-              <Textarea {...register('description')} />
-              <FormErrorMessage>
-                {errors?.description?.message}
-              </FormErrorMessage>
+              <Textarea
+                {...register('description', {
+                  required: 'Please enter a payment description',
```

Comment:
Let's revisit this with Stacey on the definition of 'Name' vs 'Description' and whichever is optional/required